lang/uk.json

```diff
@@ -1929,6 +1935,7 @@
   "TierPage.ContributorsDescription": "Join us in contributing to this tier!",
   "TierPage.Share.title": "Допоможіть {collective} досягти їхньої мети!",
   "tiers.about": "Set up contribution tiers",
+  "Tiers.CryptoTierDescription": "Enabling this will add a tier to accept Crypto donations through {theGivingBlockLink}.",
   "Tiers.CustomTierDescription": "A default tier that enables freely customizable contributions, so people can set thier own amount and frequency without limitations. You cannot change the settings or description of this tier, but you can disable it.",
```

Comment:
For some reason, this file is still outdated.

Reply:
I've updated the lang files, now it should be good. Now sure how this happened, maybe I didn't update it the first time. 🤔